mk/man.mk

```diff
@@ -29,11 +29,8 @@ HELP2MAN_ARGS = -N --section 8 --name "Part of the Pacemaker cluster resource ma
 # help. Putting the real tool in MAN8DEPS helps detect when the wrapped help
 # needs updating.
 #
-# If a ".inc" file exists, the tool has been converted to use glib for
-# argument parsing, otherwise it still uses the libcrmcommon functions.
-#
-# @TODO Drop MAN8DEPS once we've converted all tools to libpacemaker API calls
-#       and all wrappers to C code.
+# @TODO Drop MAN8DEPS once we've moved all tool functionality to libpacemaker,
+#       and all wrappers are converted from shell scripts to C code.
```

Comment:
Do we still nee the `else` conditions below?

Reply:
no

Reply:
I tried dropping them.
```
Note: Writing pacemaker-fenced.7
chmod a+x /home/reid/git/pacemaker/daemons/fenced/fence_watchdog
PATH=/home/reid/git/pacemaker/daemons/fenced:$PATH /home/reid/git/pacemaker/daemons/fenced/fence_watchdog --version >/dev/null
PATH=/home/reid/git/pacemaker/daemons/fenced:$PATH /home/reid/git/pacemaker/daemons/fenced/fence_watchdog --help-all >/dev/null
Parse error: option --help-all not recognized
Please use '-h' for usage
make[3]: *** [Makefile:1181: fence_watchdog.8] Error 1
rm pacemaker-fenced.dbook pacemaker-fenced.xml
make[3]: Leaving directory '/home/reid/git/pacemaker/daemons/fenced'
make[2]: *** [Makefile:524: all-recursive] Error 1
make[2]: Leaving directory '/home/reid/git/pacemaker/daemons'
make[1]: *** [Makefile:1129: core] Error 1
make[1]: Leaving directory '/home/reid/git/pacemaker'
make: *** [GNUmakefile:40: build] Error 2
```

Reply:
Ah, I didn't realize the daemon metadata options went through there

Reply:
I don't think they do -- that's for `fence_watchdog`

Reply:
The original point of the .inc files was that help2man had some difficulties processing the help output of tools that had been converted to use glib for command line processing.  So, the .inc file added some stuff to make the man pages look close to the same before and after that conversion.  See 86b02244bae30bbcf2a1d2b931bf98ff43b7c5df.

Basically, any tool that's using glib for command line processing should have a .inc file.  Once that is the case and everything is using glib for command line handling, we can get rid of the else branch since "<tool> --help" will be pretty useless.

Reply:
`fence_watchdog` is a python script though and will never use GLib

Reply:
Don't know if that was what Ken meant but cmdline-usage of fence_watchdog isn't very useful as only intention is to provide meta-data.
--help does state that but it might make sense to strip away the rest of the help-text or reduce it to just -o, --action (with the only action advertised metadata).

Reply:
Various things could be done here.

* Does fence_watchdog even need a man page?  If not, we need to filter it out of man8_MANS.
* The existing fence_watchdog man page doesn't look much like other man pages, presumably because of the differences between python and glib command line handling.  Somewhere we have a project for converting all our python stuff to argparse, which might help.  After that, maybe it should have a .inc file (in which case it also needs to support --help-all unless the makefile changes).
* fence_watchdog could grow a --help-all option that does the exact same thing as --help, which would help in the future to simplify some of the stuff in mk/man.mk.
* It might be possible to simplify everything in that man page generating target - it might be fine to run everything through sed, for instance.